data/strings/types_strings.txt

```diff
@@ -672,42 +672,47 @@
     zh-Hant = 酒吧
 
   [type.amenity.bbq]
-    en = Picnic Site
-    af = Piekniekterrein
-    ar = مكان نزهة
-    az = Piknik sahəsi
-    cs = Piknik
-    da = Picnic
+    en = Barbecue Grill
+    af = Braai rooster
+    ar = شواية شواء
+    az = Barbekü qril
+    be = Мангал для барбекю
+    bg = Грил барбекю
+    ca = Barbacoa
+    cs = Grilování
+    da = Grill
     de = Grillplatz
-    el = Χώρος εκδρομής
-    es = Picnic
-    et = Piknikuala
-    eu = Piknika
-    fa = مکان پیک نیک
-    fi = Piknikpaikat
-    fr = Terrain de pique-nique
-    hi = बार्बीक्यू
-    hu = Piknikezőhely
-    id = Lokasi piknik
-    it = Grill pubblico
-    ja = ピクニック
-    ko = 소풍 사이트
-    mr = सहलीचे ठिकाण
-    nb = Rasteplass
-    nl = Barbecueplaats
-    pl = Pole piknikowe
-    pt = Parque de merendas
-    pt-BR = Local de piquenique
-    ro = Locații picnic
-    ru = Место для пикника
-    sk = Piknik
-    sv = Picknickplatser
-    th = เว็บไซต์ปิกนิก
-    tr = Piknik Alanı
-    uk = Пікнік
-    vi = Điểm cắm trại
-    zh-Hans = 野餐地点
-    zh-Hant = 野餐
+    el = Ψησταριά μπάρμπεκιου
+    es = Barbacoa
```

Comment:
Strange, should be "Parrilla" somehow ..

Reply:
es-MX = Parilla

Reply:
Yes. "parrilla" is ok (in es-UY it would be better "parrillero"), also "asador" or "barbacoa" is a word that should be well undestand in LATAM...  there are a lot of variants and localizations in spanish, is a rich and complex language... :-)

Reply:
Well, google translate makes "Barbecue Grill" -> "Parrilla de barbacoa". Is it good or not?

Reply:
> Well, google translate makes "Barbecue Grill" -> "Parrilla de barbacoa". Is it good or not?

Seems that Google translate has some bias and prefer North American words :-). "barbacoa" is understand and is good, but "parrilla" or "asador" are better choices.

Reply:
Ok, so is "Parrilla de barbacoa" better than "Barbacoa" here?

Reply:
No, only "Parrilla".

Reply:
added `es-MX = Parrilla`